api/src/main/scala/org/ensime/api/config.scala

```diff
@@ -25,6 +25,8 @@ case class EnsimeConfig(
   def root = rootDir
   def debugVMArgs = debugArgs
   def referenceSourceJars = referenceSourceRoots
+  def disableSourceMonitoring = true
```

Comment:
I'm going to make these `Option`. I missed that they were hardwired. That makes the for comprehension approach suggested earlier make more sense (although admittedly its a bit weird to have an `Option[Boolean]` that would never be `false`).

For a while now I've wanted to create a user-wide config file, which might work well here. But I think this is still project specific.